### PR TITLE
feat(addon-doc): improve ui for link to sample

### PR DESCRIPTION
**Pull request by splincode**
image

My idea from google project https://google.github.io/zx/getting-started

image

### CODE REVIEW

**Review by vladimirpotekhin — APPROVED** [review]

**Comment by splincode** [comment]
@nsbarsukov

File: projects/addon-doc/components/example/example.component.ts [hunk 1]
```diff
@@ -56,6 +57,8 @@ import {TuiDocExampleGetTabsPipe} from './example-get-tabs.pipe';
         TuiMapperPipe,
         NgTemplateOutlet,
         TuiDocCode,
+        TuiLink,
+        RouterLinkActive,
```
Comment (waterplea) [inline projects/addon-doc/components/example/example.component.ts:61]: Is it used? I see you did manual check for active link, would be nice to use Angular directive for that.

Comment (splincode) [reply projects/addon-doc/components/example/example.component.ts:61]: fixed

File: projects/addon-doc/components/example/example.template.html [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
-        tuiIconButton
+        routerLinkActive="t-link-active"
+        tuiLink
         type="button"
-        class="t-link-icon"
+        class="t-link"
+        [attr.title]="copy$ | async"
         [fragment]="id"
-        [iconStart]="icons.link"
+        [routerLinkActiveOptions]="{matrixParams: 'exact', queryParams: 'exact', paths: 'exact', fragment: 'exact'}"
```
Comment (waterplea) [inline projects/addon-doc/components/example/example.template.html:20]: I think it can be just `{ exact: true }`?

image

**Review by waterplea — APPROVED** [review]

File: projects/addon-doc/components/example/example.template.html (hunk 2 above)
Comment (splincode) [reply projects/addon-doc/components/example/example.template.html]: image

```
[routerLinkActiveOptions]="{matrixParams: 'exact', queryParams: 'exact', paths: 'exact', fragment: 'exact'}"
```

image

**Pull request merged**